_docs-sources/intro/environment-setup/local_installation.md

```diff
@@ -0,0 +1,17 @@
+---
+'title': 'Local Installation'
+---
+
+# Local Installation
```

Comment:
Yeah, I like it, I'll rework it that way now to see how it looks

Reply:
In fact, we probably don't need the Recommended Tools page either, since it's basically redundant with the local installation page. So, I think this is all a one-pager.

Reply:
Addressed in 1d8804e